### PR TITLE
Adding migration and ocp v3.5 workshop content

### PR DESCRIPTION
**Pull request by DLT-Solutions-JBoss**
Adding new workshops for JBoss migration activities and updated OpenShift 101 workshop with new screen shots showing v3.5 capabilities.

### CODE REVIEW

**Comment by DLT-Solutions-JBoss** [comment]
I've made the changes to remove DLT references.  Please review PR at your convenience.  Thx!

File: content/workshops/jboss-migration-toolkit/lab4-mavenize-app.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+      <version>2.5</version>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+~~~~
+
+### BOM File
```
Comment (jaredhocutt) [inline content/workshops/jboss-migration-toolkit/lab4-mavenize-app.md]: You repeat the *BOM File* heading here. Is that intentional or do both sets of content belong under the heading by the same name above?

File: content/workshops/jboss-migration-toolkit/lab5-using-jmt-web-console.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  b. Enter a name for the project and press Next.
+
+3. Add the sample application to the project.
+Specify the applications to add to this project. You can either upload applications or register a server path that contains applications.  You can can use the Choose files button to select applications, or you can drag and drop applications into the area provided. This uploads the selected applications to the server. 
+
+<img src="../images/web-add-apps.png" width="1000" />
+
+  a. Click Choose files, select the downloaded simple-sample-app.ear file, and click Next.
+
+<ALERT>
```
Comment (jaredhocutt) [inline content/workshops/jboss-migration-toolkit/lab5-using-jmt-web-console.md]: I think you meant to use `{% alert %}` shortcode

File: content/workshops/jboss_windup/_index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: Red Hat JBoss Windup Workshop
+menu:
+  main:
+    parent: workshops
+---
+
+:badges:
```
Comment (jaredhocutt) [inline content/workshops/jboss_windup/_index.md]: This is content from asciidoc and isn't valid for markdown

File: content/workshops/jboss-migration-toolkit/prerequisites.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+All of that is being handled for you and you can **focus on the content of your workshop**.
+{{% /alert %}}
+
+
+# Supported Migration Paths
+RHAMT supports application migration from several platforms to Red Hat JBoss Enterprise Application Platform (JBoss EAP). See the below table for which JBoss EAP version is currently supported by RHAMT for direct migration from your source platform. 
+
+| Source Platform | Migration to JBoss EAP 6 | Migration to JBoss EAP 7 |
+| --------------- |:------------------------:|:------------------------:|
+| Oracle Weblogic |       ![checkmark]       |       ![checkmark]       |
```
Comment (jaredhocutt) [inline content/workshops/jboss-migration-toolkit/prerequisites.md]: `![checkmark]` doesn't get rendered as an actual checkmark image, but just as the text itself.

You could instead use `<i class="fa fa-check"></i>`

File: content/workshops/jboss-migration-toolkit/lab1-welcome.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+# Story Points
+Story points are an abstract metric commonly used in Agile software development to estimate the level of effort needed to implement a feature or change.
+
+Red Hat Application Migration Toolkit uses story points to express the level of effort needed to migrate particular application constructs, and the application as a whole. It does not necessarily translate to man-hours, but the value should be consistent across tasks. 
+
+## How Story Points are Estimated in Rules
+Estimating the level of effort for the story points for a rule can be tricky. The following are the general guidelines RHAMT uses when estimating the level of effort required for a rule.
+
+| Level of Effort | Story Points  | Description                                                                                         |
```
Comment (jaredhocutt) [inline content/workshops/jboss-migration-toolkit/lab1-welcome.md]: Check the formatting of your table here. It is not getting rendered as a table by Hugo.

File: content/workshops/jboss-migration-toolkit/lab2-using-jmt-cli.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+{{% alert warning %}}
+These instructions are only when you are doing this lab outside of the controlled workshop.  The workshop environment has downloaded, installed, and configured your student environments prior to commencement of this workshop session.
+{{% /alert %}}
+
+# Set Up
+1. Install the Java SE Development Kit (JDK) version 8. We recommend using the OpenJDK or the Oracle JDK.
+
+2. Install the RHAMT CLI.
+
+  a. Download the [ZIP archive][1].
```
Comment (jaredhocutt) [inline content/workshops/jboss-migration-toolkit/lab2-using-jmt-cli.md]: I think you need to add an additional space in front of `a.` and `b.` to get it to render as being under `2.` - currently it's breaking your numbering.

File: content/workshops/jboss-migration-toolkit/lab2-using-jmt-cli.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+~~~~
+
+# Step 1. Analyze your first application
+Now that the RHAMT CLI is installed, you’re ready to try it out. We’ll show you how to run the CLI to analyze a sample WebLogic application to identify the changes necessary to migrate it to JBoss EAP 7.
+
+> <i class="fa fa-terminal"></i> Navigate to the URI provided by your instructor and login with the user/password provided
+
+1. Escalate your user privileges.
+
+~~~
```
Comment (jaredhocutt) [inline content/workshops/jboss-migration-toolkit/lab2-using-jmt-cli.md]: This needs to be indented under your numbering. It's currently breaking your numbering and being as `1.` for each step below.

In other words, it's being rendered as:
```
1. your text
your code
1. other text that should have a 2 in front
your other code
```

File: content/workshops/jboss-migration-toolkit/lab5-using-jmt-web-console.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+The web console for Red Hat Application Migration Toolkit is a web-based system that allows a team of users to assess and prioritize migration and modernization efforts for a large number of applications. It allows you to group applications into projects for analysis and provides numerous reports that highlight the results. 
+
+{{% alert warning %}}
+The following instructions are only when you are doing this lab outside of the controlled workshop.  The workshop environment has downloaded, installed, and configured your student environments prior to commencement of this workshop session.
+{{% /alert %}}
+
+# Set Up
+1. Install the Java SE Development Kit (JDK) version 8. We recommend using the OpenJDK or the Oracle JDK.
+2. Install the RHAMT web console.
+  a. Download the [ZIP archive][1].
```
Comment (jaredhocutt) [inline content/workshops/jboss-migration-toolkit/lab5-using-jmt-web-console.md]: Similar comment regarding the numbering and indenting. The rendered numbering is messed up throughout this page.

Take a look at the generated content on your local machine by running the command `hugo server`

File: themes/redhatgov/layouts/partials/nav.html [hunk 9]
```diff
@@ -12,6 +12,9 @@
       <a class="navbar-brand" href="/">
         <img src="{{ "img/brand.png" | absURL }}" alt="{{ .Site.Title }}" />
       </a>
+      <a class="navbar-brand" href="/">
```
Comment (jaredhocutt) [inline themes/redhatgov/layouts/partials/nav.html]: Remove this from the header.

**Comment by jaredhocutt** [comment]
This pull request can't be merged due to being customized for DLT. Based on the continued work on this and the customization, I'm assuming that DLT is running a copy of their site for their own purposes.

I am closing this pull request without merging. If you would like to have the additions and/or changes that you've made to workshop content please break that content off into separate pull requests for each.

**Pull request closed without merging**